app/assets/stylesheets/modules/layout.scss

```diff
@@ -30,6 +30,12 @@
   }
 }
 
+.site-navbar {
+  .search-query-form {
+    min-width: 17em;
+  }
+}
+
 .exhibit-navbar {
   .search-query-form {
     .btn-primary {
```

Comment:
A side-effect of this class name change is that this button is now cardinal red instead of the secondary beige-y color. Not sure if this is intentional?

## Before
![Screen Shot 2020-02-27 at 4 13 54 PM](https://user-images.githubusercontent.com/5402927/75499562-38fc5100-597f-11ea-879d-10e06fc13358.png)

## After
![Screen Shot 2020-02-27 at 4 33 41 PM](https://user-images.githubusercontent.com/5402927/75499560-3863ba80-597f-11ea-8685-da8ea23dcf03.png)

Reply:
I'd lean toward the beige-y color, just because the red is pretty eye-catching and in this context I think browsing the cards is still the most expected user action.

Reply:
Thanks, nice catch. I think we still should update the class name, but have it share that style (as in 7bb380f)?